dev-util/spirv-llvm-translator/spirv-llvm-translator-13.0.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+
+inherit cmake-multilib flag-o-matic llvm
+
+MY_PN="SPIRV-LLVM-Translator"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Bi-directional translator between SPIR-V and LLVM IR"
+HOMEPAGE="https://github.com/KhronosGroup/SPIRV-LLVM-Translator"
+SRC_URI="https://github.com/KhronosGroup/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="UoI-NCSA"
+SLOT="13"
+KEYWORDS="~amd64"
+IUSE="test +tools"
+
+RESTRICT="!test? ( test )"
+
+S="${WORKDIR}/${MY_P}"
+
+COMMON="sys-devel/clang:13=[${MULTILIB_USEDEP}]"
+DEPEND="${COMMON}
+		dev-util/spirv-headers"
+RDEPEND="${COMMON}"
+BDEPEND="test? ( dev-python/lit )"
```

Comment:
We may need to use python-any-r1 to ensure we have Python for tests too, right? Or is it just needing some lit executable?

Reply:
Oh, sorry, I see that now. Yeah, ignore this.